wave_matching/CMakeLists.txt

```diff
@@ -1,7 +1,8 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8.3)
 PROJECT(wave_matching)
 
-SET(CMAKE_BUILD_TYPE RELEASE)
+SET(CMAKE_CXX_FLAGS "-std=gnu++11 ${CMAKE_CXX_FLAGS}")
```

Comment:
`c++11`

Reply:
do you mean change gnu++11 to c++11?

Reply:
Yes. Actually it is already set in top-level CMakeLists.txt so can remove